readthedocs/oauth/admin.py

```diff
@@ -19,14 +19,30 @@ class RemoteRepositoryAdmin(admin.ModelAdmin):
     raw_id_fields = ('users',)
```

Comment:
I think we need to update these `raw_id_fields` as well, because is timing out when accessing to a specific `RemoteRepository`. Not sure which ones we need to add here.